src/utils/stringUtils.spec.ts

```diff
@@ -29,3 +30,29 @@ describe('stringToBool()', () => {
     expect(stringToBool('TrUe')).toEqual(true);
   });
 });
+
+describe('truncateOnMiddle()', () => {
```

Comment:
It would have been nice to have more tests, in order to verify (and explain/document) things like the output string having the right length even including ellipsis, and other little details like that, but I don't think it's necessary to change.